Make consistent in two SimulatorClient in Preview

ianychoi opened the pull request:
BinaryMessageInbox in Preview/Core/SimulatorClient.uno is no longer
referenced and Preview/App/Fuse.Simulator/SimulatorClient.uno
needs to have the same structure by commenting _alsoReceieveMessagesFromPipe.

Closes: #13

This PR includes:
- [ ] Change log
- [ ] Tests

kusma commented on Source/Preview/App/Fuse.Simulator/SimulatorClient.uno:
```diff
@@ -113,7 +113,7 @@ namespace Fuse.Simulator
 		readonly Thread _readWorker;
 		readonly Thread _writeWorker;
-		readonly IDisposable _alsoReceieveMessagesFromPipe;
+		//readonly IDisposable _alsoReceieveMessagesFromPipe;
```
Just delete the declaration, no point in keeping it commented out...

ianychoi replied:
Done - thanks!

kusma commented on Source/Preview/App/Fuse.Simulator/SimulatorClient.uno:
```diff
@@ -205,7 +203,7 @@ namespace Fuse.Simulator
 			_running = false;
 			
-			_alsoReceieveMessagesFromPipe.Dispose();
+			//_alsoReceieveMessagesFromPipe.Dispose();
```
Just delete the line, no point in keeping it commented out...

ianychoi replied:
Also done - thank u for your review!

kusma commented:
OK, so I guess the next question is, should this be done against release-1.9 or master? If the latter, should it be back-ported to release-1.9?

lorents commented:
If we put it on 1.9 we're sure that if someone builds the app for use with 1.9 it actually works

lorents commented:
(Generally I'd prefer rebase this on the affected release branch and merge that to master than having two commits for this fix, if the latter is what you mean by backporting it)

BeeWarloc commented:
@ianychoi Thanks for the contribution. 👍

Yes, this should probably target the `release-1.9` branch. There's no need for creating new installers though, as this only affects the preview app released on the app stores.

I tried building the preview app for iOS, and I can confirm that it fixes the compilation error. However, I do not have any test devices at hand right now, so I'm not currently able to test the resulting artifact.

I'm also not able to build this for Android right now, due to some unrelated weird Java problem on my laptop. 

I will try to look more into this tomorrow.

ianychoi commented:
@BeeWarloc It's my pleasure and thanks a lot for dealing with fuse-studio stuff including preview app.

I tested with 1) current iOS preview app does not work with Fuse 1.9 and 2) My locally built iOS & Android preview apps worked well with Fuse 1.9 and it worked fine. (Note that I just changed iOS bundle identifier since it seems that I cannot use `com.fusetools.fusepreview` )

kusma commented:
@ianychoi: Would you mind rebasing this on release-1.9, and update the merge-base for the PR?

ianychoi commented:
@kusma I changed the base branch to `release-1.9` and updated my forked master branch with one commit of this pull request based from `release-1.9`. Not sure whether my method is general or not (I should have created a local branch rather than just using `master` local branch) but this would be identical with your suggestion I think :)

kusma commented:
@ianychoi: Thanks, that's perfect.

Yeah, generally sending PRs from designated branches makes it a bit easier to keep things sane, but this works also :)
